### PR TITLE
M5 stamp s3 cardputer build

### PR DESCRIPTION
**Pull request by jamesjnadeau**
This is my working build of circuit python for the M5 stack cardputer.

I have updated the USB parameters per https://github.com/espressif/usb-pids

Thanks for your time and for making this pretty easy to get going, your docs are fabulous!

### CODE REVIEW

**Comment by jamesjnadeau** [comment]
not sure what was up with the .pot file change... I can remake without that...

File: ports/espressif/boards/m5stack_cardputer/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,11 @@
+USB_VID = 0x303A
+USB_PID = 0x811A
+USB_PRODUCT = "M5STACK CoreS3 - CircuitPython"
+USB_MANUFACTURER = "M5STACK"
```
Comment (tannewt) [inline ports/espressif/boards/m5stack_cardputer/mpconfigboard.mk:4]: These look like the values for a different board. Please update.

**Review by tannewt — CHANGES_REQUESTED** [review]
Yes, please remove the .pot file too. It will be autoupdated separately. Thanks for the new board def!

**Comment by jamesjnadeau** [comment]
I'm going to close this and resubmit

**Pull request closed without merging**